lib/cstruct_cap.mli

```diff
@@ -0,0 +1,88 @@
+type 'a rd = < rd: unit; .. > as 'a
+type 'a wr = < wr: unit; .. > as 'a
+type 'a aligned = < aligned: unit; .. > as 'a
```

Comment:
This is not used anywhere, what is its purpose? I'd drop it in a first revision, and add alignment at a later point